crates/cli/src/cloud/mod.rs

```diff
@@ -123,10 +125,31 @@ impl CloudStartCommand {
             ..ColorfulTheme::default()
         };
 
-        let workspaces = get_user_workspaces(&auth_config.access_token, svm_gql_url)
+        let mut workspaces = fetch_svm_workspaces(&auth_config.access_token, svm_gql_url)
             .await
             .map_err(|e| format!("failed to get available workspaces: {}", e))?;
 
+        if workspaces.is_empty() {
```

Comment:
Do you think we should make this block part of the get user workspaces function?

Or at least making hard refresh a function somewhere in the cloud crate?

Also, why should this make any difference? We've just made a request with a refreshed access token, why would logging in again help?

Reply:
I'm not sure, there seems to be a cache being evicted backend side when regenerating a new access token.

Reply:
> Do you think we should make this block part of the get user workspaces function?

I'd love to get the cloud api fixed asap.

Reply:
So are we making this request just because something isn't working backend?

Reply:
Hasura side.

Reply:
So is this a bandaid? I'm cool with merging a bandaid, but maybe we can get a comment and an issue in linear saying that?

Reply:
I created an entry in Linear!